src/handler-builder.ts

```diff
@@ -35,6 +35,15 @@ import * as remoteConfig from './providers/remoteConfig';
 import * as storage from './providers/storage';
 import * as testLab from './providers/testLab';
 
+/**
+ * The `HandlerBuilder` class facilitates the writing of functions by producers
+ * of Firebase Extensions and developers who want to use the gcloud CLI or
+ * Google Cloud Console to deploy their functions.
+ *
+ * **Do not use `HandlerBuilder` when writing normal functions for deployment via
+ * Cloud Functions for Firebase.** For normal purposes, use
```

Comment:
`deployment via Firebase CLI` is more understandable.

Reply:
Done, thanks!